src/PactNet/RequestBuilder.cs

```diff
@@ -241,8 +241,16 @@ IRequestBuilderV3 IRequestBuilderV3.WithJsonBody(dynamic body, JsonSerializerSet
         /// <param name="contentType">Content type override</param>
         /// <returns>Fluent builder</returns>
         IRequestBuilderV3 IRequestBuilderV3.WithJsonBody(dynamic body, JsonSerializerSettings settings, string contentType)
-            => this.WithJsonBody(body, settings, contentType);
-
+            => this.WithJsonBody(body, settings, contentType);
+
+        /// <summary>
+        /// Set a body which is multipart/form-data but contains only one part, which is a file upload
+        /// </summary>
+        /// <param name="filePath">Path to the file being uploaded</param>
+        /// <returns>Fluent builder</returns>
+        IRequestBuilderV3 IRequestBuilderV3.WithMultipartSingleFileUpload(string filePath)
+            => this.WithMultipartSingleFileUpload(filePath, "multipart/form-data", "file");

```

Comment:
You can't hard-code `"file"` here because that influences the name of the part on the provider side, which might be important.